voice/make-an-outbound-call.py

```diff
@@ -0,0 +1,25 @@
+import os
+from os.path import join, dirname
+from dotenv import load_dotenv
+
+dotenv_path = join(dirname(__file__), '../.env')
+load_dotenv(dotenv_path)
+
+APPLICATION_ID = os.environ.get('APPLICATION_ID')
+APPLICATION_PRIVATE_KEY_PATH = os.environ.get('APPLICATION_PRIVATE_KEY_PATH')
+TO_NUMBER = os.environ.get('TO_NUMBER')
+NEXMO_NUMBER = os.environ.get('NEXMO_NUMBER')
+APPLICATION_PRIVATE_KEY = open('private.key','r').read()
```

Comment:
No need to read the key in - `NexmoClient` will automatically detect a key vs a path, and read in the file if necessary.